backend/manager/modules/vdsbroker/src/test/java/org/ovirt/engine/core/vdsbroker/monitoring/VirtMonitoringStrategyTest.java

```diff
@@ -172,6 +192,22 @@ public void testNeedToProcessHardwareCapsTrue() {
         newVds.setCpuFlags("flag2");
         assertTrue(virtStrategy.processHardwareCapabilitiesNeeded(oldVds, newVds));
     }
+    private  ImageTransferDao mockImageTransferDao() {
+        ImageTransferDao mock = mock(ImageTransferDao.class);
+        List<ImageTransfer> imageTransfers = new ArrayList<>();
+        List<ImageTransfer> imageTransfers2 = new ArrayList<>();
+        List<ImageTransfer> imageTransfers3 = new ArrayList<>();
+        ImageTransfer imageTransfer = new ImageTransfer();
+        ImageTransfer imageTransfer2 = new ImageTransfer();
+        imageTransfer.setPhase(ImageTransferPhase.FINISHED_SUCCESS);
+        when(mock.getByVdsId(vdsId3)).thenReturn(imageTransfers);
```

Comment:
Suggestions to improve readability:
Better rename `imageTransfer` & `imageTransfer2` to `imageTransferFinished` & `imageTransferWip`
Instead of `imageTransfers` simply use `Collections.emptyList()`.
And better use `Collections.singletonList` or `Arrays.asList()` in order to create the `List`s (`imageTransfers2`& `imageTransfers3`) in-place, since the names are not  helpful anyway, so no need in them.

Reply:
done please review